msalautomationapp/src/main/java/com/microsoft/identity/client/msal/automationapp/sdk/MsalAuthResult.java

```diff
@@ -42,4 +46,12 @@ public MsalAuthResult(@NonNull final IAuthenticationResult authenticationResult)
     public MsalAuthResult(@NonNull final Exception exception) {
         super(exception);
     }
+
+    public static void verifyATForPop(@NonNull final String shr) throws ServiceException {
+        Map<String, ?> tokens = IDToken.parseJWT(shr);
```

Comment:
```suggestion
        final Map<String, ?> tokens = IDToken.parseJWT(shr);
```

Reply:
https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1643 This PR has the change suggested